src/Raven.Server/Program.cs

```diff
@@ -102,6 +103,9 @@ public static int Main(string[] args)
                             if (CommandLineSwitches.PrintServerId)
                                 Console.WriteLine($"Server ID is {server.ServerStore.GetServerId()}.");
 
+                            if (CommandLineSwitches.PrintRuntimeSettings)
```

Comment:
Is there a reason not to put this by default?
Otherwise, we'll need to restart the service for this.
Let us add this to the info command as well.